src/main/java/core/basesyntax/dao/impl/MessageDaoImpl.java

```diff
@@ -12,21 +15,65 @@ public MessageDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Message create(Message entity) {
-        return null;
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(entity);
```

Comment:
```suggestion
            session.save(entity);
```
https://mate-academy.github.io/jv-program-fulltime/03_hibernate/checklist/hibernate_02.html

Reply:
done